### PR TITLE
Add Kmtronic

### PR DESCRIPTION
**Pull request by dgomes**
## Proposed change

Adds the KMTronic integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/41682
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1926
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/kmtronic.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: KMTronic
+description: Integrate KMTronic devices
+ha_category:
+  - Switch
+ha_release: 0.119
+ha_codeowners:
+  - '@dgomes'
+ha_iot_class: Local Push
+ha_domain: kmtronic
+featured: false
```
Comment (frenck) [inline source/_integrations/kmtronic.markdown]:
```suggestion
```

File: source/_integrations/kmtronic.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ha_config_flow: true
+---
+
+Integrate [KMtronic devices](https://www.kmtronic.com/) into Home Assistant.
```
Comment (frenck) [inline source/_integrations/kmtronic.markdown]: Maybe change this part into what `kmtronic` actually is? As in, an introduction to the integration.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/kmtronic.markdown (hunk 2 above)
Comment (dgomes) [reply source/_integrations/kmtronic.markdown]: Thanks for the feedback!

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍
⚠️ The ha_release might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

File: source/_integrations/kmtronic.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: KMTronic
+description: Integrate KMTronic devices
+ha_category:
+  - Switch
+ha_release: 0.119
```
Comment (bdraco) [inline source/_integrations/kmtronic.markdown]:
```suggestion
ha_release: 2021.3
```

File: source/_integrations/kmtronic.markdown [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ha_iot_class: Local Push
+ha_domain: kmtronic
+ha_config_flow: true
+---
+
+Integrate [KMtronic devices](https://www.kmtronic.com/) into Home Assistant. KMtronic offers a large range of IP connected relays that can be controlled over your local area network. This integration maps each of the relays as a separate device, enabling you to control various areas of your home (e.g. using each relay to control different garden sprinkler areas, or controlling electrical distribution of your house from a central board).
+
+## Configuration
+
+To add a KMtronice device to your installation, make sure they are connected to your home network first with a fixed IP. Next, go to **Configuration** >> **Integrations** in the UI. Click the button with `+` sign on the integrations page and from the list of integrations, select **KMtronic** and insert the IP address, Username and Password of your KMtronic device. The integration will detect how many relays your device has, and will create a switch entity for each of the relays.
```
Comment (bdraco) [inline source/_integrations/kmtronic.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

You'll need to rebase for that to work.

**Comment by homeassistant** [comment]
Hi @slydiman, @loganmarchione, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#15953).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/kmtronic.markdown (hunk 4 above)
Comment (dgomes) [reply source/_integrations/kmtronic.markdown]: done! thanks!

File: source/_integrations/kmtronic.markdown [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  - Switch
+ha_release: 2021.3 
+ha_codeowners:
+  - '@dgomes'
+ha_iot_class: Local Push
+ha_domain: kmtronic
+ha_config_flow: true
+---
+
+Integrate [KMtronic devices](https://www.kmtronic.com/) into Home Assistant. KMtronic offers a large range of IP connected relays that can be controlled over your local area network. This integration maps each of the relays as a separate device, enabling you to control various areas of your home (e.g. using each relay to control different garden sprinkler areas, or controlling electrical distribution of your house from a central board).
```
Comment (frenck) [inline source/_integrations/kmtronic.markdown]:
```suggestion
Integrate [KMtronic devices](https://www.kmtronic.com/) into Home Assistant. KMtronic offers a large range of IP-connected relays that can be controlled over your local area network. This integration maps each of the relays as a separate device, enabling you to control various areas of your home (e.g. using each relay to control different garden sprinkler areas, or controlling the electrical distribution of your house from a central board).
```

File: source/_integrations/kmtronic.markdown [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+## Configuration
+
```
Comment (frenck) [inline source/_integrations/kmtronic.markdown]: The header is automatically inserted by the include below (and thus now duplicate)

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @dgomes 👍

**Pull request merged**